src/autoscaler/db/sqldb/scheduler_sqldb.go

```diff
@@ -82,3 +83,23 @@ func (sdb *SchedulerSQLDB) GetActiveSchedules() (map[string]*models.ActiveSchedu
 	return schedules, nil
 
 }
+
+func (sdb *SchedulerSQLDB) SynchronizeActiveSchedules(appIdMap map[string]bool) error {
```

Comment:
for future reference, a set in Go is map[string]struct{}. The size of a struct{} in Go is 0.